src/Sylius/Behat/Page/Admin/Currency/UpdatePageInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Currency;
+
+use Sylius\Behat\Page\Admin\Crud\UpdatePageInterface as BaseUpadatePageInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+interface UpdatePageInterface extends BaseUpadatePageInterface
+{
+    public function enable();
+    public function disable();
+
+    /**
+     * @param string $exchangeRate
+     */
+    public function changeExchangeRate($exchangeRate);
+
+    /**
+     * @param string $exchangeRate
+     *
+     * @return bool
+     */
+    public function assertExchangeRate($exchangeRate);
```

Comment:
Same as above, ``getExchangeRate`` is more generic, it allows us to get the value and we can make the assertion about the value in the context.